internal/accounts/create.go

```diff
@@ -134,3 +147,301 @@ func create(
+			nil,
+		)
+		if err != nil {
+			return nil, err
+		}
+		log.StopProgress()
+
+		log.Info("\nPlease note, that the newly created account will only be available until you keep the emulator service up and running, if you restart the emulator service all accounts will be reset. If you want to persist accounts between restarts you must use the '--persist' flag when starting the flow emulator.")
+
+		address = account.Address
+	} else {
+		var link string
+		switch selectedNetwork {
+		case config.DefaultTestnetNetwork():
+			log.Info(fmt.Sprintf("%s We will be creating the account on testnet faucet website (%s), which has been prefilled with generated keys", output.WarningEmoji(), util.TestnetFaucetHost))
+			log.Info("\nPlease follow the steps: \n 1. Fill in the captcha, \n 2. Click on 'Create Account' button.\n")
+			output.NextStepPrompt()
+			link = util.TestnetFaucetURL(key.PublicKey().String(), crypto.ECDSA_P256)
+		case config.DefaultMainnetNetwork():
+			log.Info(fmt.Sprintf("%s We will be creating the account on Flow Port website (%s), which has been prefilled with generated keys", output.WarningEmoji(), util.FlowPortUrl))
+			log.Info("\nPlease follow the steps: \n 1. Click on 'Submit' button, \n 2. Connect existing Blocto or create a new account first, \n 3. Click on confirm, \n 4. Click on approve. \n")
+			output.NextStepPrompt()
+			link = util.MainnetFlowPortURL(key.PublicKey().String())
+		}
+		log.StartProgress("Waiting for an account to be created, please finish all the steps in the browser...\n")
+		time.Sleep(time.Second * 2)
+		err := util.OpenBrowserWindow(link)
+		if err != nil {
+			return nil, err
+		}
+
+		addr, err := getAccountCreatedAddressWithPubKey(service, key.PublicKey(), startHeight)
+		if err != nil {
+			return nil, err
+		}
+		address = *addr
+
+		log.StopProgress()
+	}
+
+	onChainAccount, err := service.Accounts.Get(address)
+	if err != nil {
+		return nil, err
+	}
+
+	account, err := flowkit.NewAccountFromOnChainAccount(name, onChainAccount, key)
+	if err != nil {
+		return nil, err
+	}
+	log.Info(fmt.Sprintf("\n %s Successfully created account on %s", output.OkEmoji(), networkName))
+	output.NextStepPrompt()
+
+	log.Info(fmt.Sprintf("\n %s Saving created account in flow.json", output.WarningEmoji()))
+	log.StartProgress("")
+	time.Sleep(time.Second * 3)
+	log.StopProgress()
+
+	err = saveAccount(loader, state, account, selectedNetwork, saveToGitIgnore)
+	if err != nil {
+		return nil, err
+	}
+	log.Info(fmt.Sprintf("%s Successfully saved account in flow.json", output.OkEmoji()))
+	if selectedNetwork != config.DefaultEmulatorNetwork() {
+		log.Info(fmt.Sprintf("%s private key for %s successfully saved in %s", output.OkEmoji(),
+			name, privateJsonFileName))
+		if saveToGitIgnore {
+			log.Info(fmt.Sprintf("%s %s added to .gitignore", output.OkEmoji(), privateJsonFileName))
+		}
+	}
+
+	log.StartProgress("")
+	time.Sleep(time.Second * 5)
+	log.StopProgress()
+	log.Info("\n------------------------")
```

Comment:
I don't like this separator, it's never used in the CLI.

Reply:
addressed